### DISCUSSION TITLE
Não consigo visualizar na versão do Portugol Web as funções de cada biblioteca no menu Ajuda.

### DISCUSSION THREAD

**Discussion by rcepgs2022** [dgadelha/Portugol-Webstudio#245] [Dúvidas / Perguntas]
Sou professora e alguns dos meus alunos só conseguem usar o Portugol Webstudio. Observei que quando acesso a página Ajuda - Bibliotecas, não tenho a opção de visualizar as funções de cada biblioteca, como acontece na versão Portugol Studio.
Será que estou acessando errado?

![portugol_webstudio_funcoes_bibliotecas_nao_aparecem](https://github.com/dgadelha/Portugol-Webstudio/assets/119897018/89c50d47-385b-4338-802e-59bfcd175ba6)

![portugol_studio_funcoes_bibliotecas](https://github.com/dgadelha/Portugol-Webstudio/assets/119897018/5470897f-1a62-41e0-82e3-9acc5371863b)

**Comment by dgadelha** [comment]
Olá!

Infelizmente ainda não temos essa seção de Bibliotecas na nossa aba de Ajuda, mas é algo que quero implementar no futuro.

Vou criar uma issue a partir desta discussão para lembrar de que isso precisa ser feito.

**Reply by dgadelha** [reply]
Criada #259, estarei encerrando a discussão por se tratar de um problema.

**Comment by FigFelipe** [comment]
Poderiam disponibilizar o link da biblioteca de documentação que está funcionando? Grato.

**Reply by dgadelha** [reply]
Olá @FigFelipe! Se a seção Ajuda não estiver funcionando para você, pode abrir uma _issue_ aqui relatando o problema? Segue o link: https://github.com/dgadelha/Portugol-Webstudio/issues/new/choose
